python/enki.cpp

```diff
@@ -189,10 +189,11 @@ static World::GroundTexture loadTexture(const std::string& fileName)
 	QImage gt(QGLWidget::convertToGLFormat(QImage(fileName.c_str())));
 	
 	#if QT_VERSION >= QT_VERSION_CHECK(4,7,0)
-	return World::GroundTexture(gt.width(), gt.height(), (const uint32_t*)gt.constBits());
+	const uint32_t* bits = (const uint32_t*)gt.constBits();
 	#else
-	return World::GroundTexture(gt.width(), gt.height(), (uint32_t*)gt.bits());
+	unint32_t* bits = (uint32_t*)gt.bits();
```

Comment:
It seems there is a typo, I do not think the type you want is `unint32_t*` but rather `uint32_t*`.